distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1127,15 +1128,15 @@ type Column
        > Example
          Remove leading and trailing spaces from cells.
 
-             column.replace "^\s*(.*?)\s*$" "$1" use_regex=True
+             column.replace "^\s*(.*?)\s*$".to_regex "$1"
 
        > Example
          Replace texts in quotes with parentheses.
 
-             column.replace '"(.*?)"' '($1)' use_regex=True
+             column.replace '"(.*?)"'.to_regex '($1)'
     replace : Text | Column -> Text | Column -> Case_Sensitivity -> Boolean -> Boolean -> Column
```

Comment:
I think that we want the type signatures to be consistent between in-memory and Database, so let's make Database counterparts also take a `Regex` whenever in-memory accepts it.

We can later throw `Unsupported_Database_Operation` if the regex is actually provided, but it should be that instead of a type error.

```suggestion
    replace : Text | Regex | Column -> Text | Regex | Column -> Case_Sensitivity -> Boolean -> Boolean -> Column
```

Reply:
Done.